src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/client/payhub/PayHubClient.java

```diff
@@ -16,6 +17,7 @@
     name = "pay-hub-api",
     url = "${pay-hub.api.url}"
 )
+@Profile({"integration", "!functional"})
```

Comment:
please check if profiles need to to be added here

Reply:
Given this is application code, having `integration` profile here looks suspicious.

Reply:
Oh, yes, removed